documentation/src/main/asciidoc/topics/code_examples/GlobalAuthorizationConfiguration.java

```diff
@@ -1,4 +1,4 @@
-GlobalRoleConfigurationBuilder global = new GlobalConfigurationBuilder();
+GlobalConfigurationBuilder global = new GlobalConfigurationBuilder();
   global.security().authorization().enable()
```

Comment:
@dvagnero these are implicit roles. is this a good idea to put them as an example?

Reply:
Yes, it does not make much sense to define these. I'll change this

Reply:
@karesti updated